packages/presentation/pubspec.yaml

```diff
@@ -15,10 +15,13 @@ dependencies:
 
   flutter_bloc: ^8.1.1
   equatable: ^2.0.5
+  get_it: ^7.2.0
```

Comment:
you don't need it here, it will be pulled from the domain

Reply:
corrected